File: codeforces/A/cf731A.cpp
#include <iostream>
#include <string>
#include <algorithm>
#include <cstring>
#include <cmath>
#define ll long long 

int main() {
    std::string phrase = "abcdefghijklmnopqrstuvwxyz";
    std::string inp;
    ll step=0;
    std::cin >> inp;
    char pivot = 'a';
    for (char c: inp) {
        double x = (double) phrase.find(pivot);
        double y = (double) phrase.find(c);
        step += std::min(abs(x-y),abs(inp.size() - x - y));
        pivot = c;
    }

    std::cout << step << std::endl;
    return 0;
}

File: codeforces/A/cf731A.py
s=input("")
lex="abcdefghijklmnopqrstuvwxyz"
pivot='a'
count=0
for c in s:
    count += min(abs(lex.index(pivot)-lex.index(c)),len(lex)-abs(lex.index(pivot)-lex.index(c)))
    pivot=c
print(count)
